migrations/20180213214150_create_sessions_table.js

```diff
@@ -0,0 +1,18 @@
+exports.up = (knex, Promise) =>
```

Comment:
sorry, I dont have context into where this file comes from. is it exported from schema, or is manual create. can you speak to naming?

Reply:
The file is auto generated by running `npx knex migrate:make file_name`.  As the command suggests, it is generated by `knex`